docs/kiwix-serve.rst

```diff
@@ -397,7 +397,41 @@ under ``/skin``.
 ``/suggest``
 ------------
 
-Blablabla
+**Usage:**
+
+  ``/suggest?content=ZIMID[&term=QUERY][&count=N][&start=S]``
+
+Returns suggestions (in JSON format) for a text string that is assumed to be a
+partially typed search for a page inside a particular ZIM file.
+
+Suggestions are obtained as matches of the query text against the page titles
+in the ZIM file using the title index database generated during the creation of
+the ZIM file. If the ZIM file doesn't contain a title index then no suggestions
+are produced.
```

Comment:
If the ZIM file doesn't contain a title index suggestions are created by listing article's titles starting with `QUERY`

Reply:
Fixed